doc/user/content/sql/create-sink.md

```diff
@@ -169,7 +169,7 @@ You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
 #### Enabling topic reuse after restart
 
-{{< experimental v0.8.2 />}}
+{{< beta />}}
```

Comment:
Let's specify `{{< beta v0.8.4 />}}` so users know the minimum version that will work. (I'm not sure users upgrade as reliably as we do internally, if the current version is working for them.)

Reply:
That's a very good point! I'm not sure the `beta` tag allows specifying a version, though. I looked at existing instances in the doc and none had one.

You think I should add that feature for the beta tag and then use it here?

Reply:
You can try doing a local build first to see if it's already supported (`hugo serve -D`), but yes, if it's not there, adding the feature would be great!

Reply:
I updated the beta shortcode and added the version. :+1: